server/exercise/src/main/scala/com/eigengo/lift/exercise/classifiers/workflows/ClassificationAssertions.scala

```diff
@@ -9,7 +9,7 @@ object ClassificationAssertions {
    * Named gesture matches with probability >= `matchProbability`
    */
   def Gesture(name: String, matchProbability: Double, sensor: SensorDataSourceLocation): GroundFact =
-    new GroundFact(name, Some((matchProbability, sensor))) {
+    new GroundFact(name, matchProbability, sensor) {
```

Comment:
It does occur to me that you might be referring to the fact that one needs to define extra accessor methods/vals in order to gain access to the contents of the `values` argument?

We get away with it here as, thus far, we've only cared about case class equality.

If we need to get convenient access to `GroundFact` arguments (in `values`), then yes we'll need to add in some extra defs/vals for that purpose. 

Pattern matching is currently good, but again it may be necessary to override `unapply` in order to ensure more case class like behaviour.